guides/v2.1/cloud/basic-information/starter-develop-deploy-workflow.md

```diff
@@ -177,7 +152,7 @@ For instructions, see [Install optional sample data]({{ page.baseurl }}/cloud/ho
 ![Install optional sample data]({{ site.baseurl }}/common/images/cloud_workflow-sample-data.png)
 
 ### Optional: Pull production data {#prod-data}
-We recommend adding all of your products, catalogs, site content, and so on (not configurations) directly in Production. By adding this data in Production, you immediately update prices, coupons, inventory stock, strategize your sales and future offerings, and much more for your customers. This data does not include extension configurations. You will set those in your development branch on your local.
+We recommend adding all of your products, catalogs, site content, and so on (not configurations) directly in Production. By adding this data in Production, you immediately update prices, coupons, inventory stock, strategize your sales and future offerings, and much more for your customers. This data does not include extension configurations. You can configure extensions in your local development branchThe {{site.data.var.ece}} includes a single Git repository with a Production master that can be branched to create Staging and Integration environments for testing and development work. See [Starter architecture]({{ page.baseurl }}/cloud/basic-information/starter-architecture.html) for an overview.
```

Comment:
There is a broken sentence in here with no punctuation. Do we still recommend this working directly on Production? I think there is a cut & paste in error here, because I have a sense of deja vu.